settings.h

```diff
@@ -20,6 +20,12 @@ e-mail   :  support@circuitsathome.com
 #include "macros.h"
 
 ////////////////////////////////////////////////////////////////////////////////
+// SPI Configuration
+////////////////////////////////////////////////////////////////////////////////
+#define USB_SPI SPI
```

Comment:
Please add ```ifndef``` around this. I.e:
```C++
#ifndef USB_SPI
#define USB_SPI SPI
#endif
```
This allows users to override it via a build flag.